content/source/docs/cloud/sentinel/import/tfplan-v2.html.md

```diff
@@ -313,6 +313,8 @@ and data sources within this plan.
 This includes all resources that have been found in the configuration and state,
 regardless of whether or not they are changing.
 
+~> When [resource targeting](/docs/commands/plan.html#resource-targeting) is in effect, the `resource_changes` collection will only include the resources specified as targets for the run. This may lead to unexpected outcomes if a policy expects a resource to always exist. To prohibit targeted runs altogether, use [`tfrun.target_addrs`](./tfrun.html#value-target_addrs).
```

Comment:
Maybe just "use tfrun.target_addrs" is vague?

'Ensure `tfrun.target_addrs` is nil or empty'? might not be better?

Reply:
I like it!